src/validators/index.ts

```diff
@@ -0,0 +1,24 @@
+import { NextFunction, Request, Response } from 'express'
+import { validateSatelliteIdRoute } from './satelliteIdValidator'
+import { validateSatelliteNameRoute } from './satelliteNameValidator'
+
+class InvalidPathError extends Error {
+    constructor(message: string) {
+        super(message)
+        this.name = 'InvalidPathError'
+    }
+}
+
+const validators: Record<string, typeof validateSatelliteIdRoute> = {
+    '/two-line-elements': validateSatelliteIdRoute,
+    '/satellite/by-id': validateSatelliteIdRoute,
+    '/satellite/by-name': validateSatelliteNameRoute
+}
+
+//ReturnType<typeof validateSatelliteIdRoute> | InvalidPathError
```

Comment:
Remove this commented-out line. You can always look through Git history to see old code. By removing unused code, you will keep your codebase clean.